cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/GetRegionsV2Handler.java

```diff
@@ -38,17 +38,15 @@ public void accept(Event<GetPlatformRegionsRequestV2> getRegionsRequestEvent) {
             CloudPlatformVariant cloudPlatformVariant = new CloudPlatformVariant(
                     Platform.platform(request.getExtendedCloudCredential().getCloudPlatform()),
                     Variant.variant(request.getVariant()));
-            CloudRegions cloudRegions = null;
-            try {
-                cloudRegions = cloudPlatformConnectors.get(cloudPlatformVariant)
-                        .platformResources().regions(request.getCloudCredential(), Region.region(request.getRegion()), request.getFilters());
-            } catch (Exception e) {
-                cloudRegions = new CloudRegions();
-            }
-            GetPlatformRegionsResultV2 getPlatformRegionsResultV2 = new GetPlatformRegionsResultV2(request, cloudRegions);
+
+            GetPlatformRegionsResultV2 getPlatformRegionsResultV2 = null;
```

Comment:
Define as null is not necessary, would be nice to declare before first ussage.

Reply:
Nice catch! :)